master/docs/tutorial/docker.rst

```diff
@@ -64,12 +64,12 @@ Building and running Buildbot
   docker-compose up
 
 
-You should now be able to go to http://localhost:8010 and see a web page similar to:
```

Comment:
We should rather fix the docker config to use 8010, as it is our real default port